### ISSUE TITLE
🐛 [Samba NAS] external disk fails to mount because credentials file does not exist

### ISSUE THREAD

**Issue by tr4nt0r** [dianlight/hassio-addons#170]
### Addon

SambaNAS

### Description

My external NTFS formated SSD sometimes gets corrupt directories and i have to remove and and fix it on my windows machine. After reconnecting it to Home Assistant and restarting SambaNAS it fails to remount the drive and shuts down immediately.

I found the following errors in the supervisor and host error logs:

supervisor:

> ERROR (MainThread) [supervisor.mounts.mount] Reloading SSD did not succeed. Check host logs for errors from mount or systemd unit mnt-data-supervisor-mounts-SSD.mount for details.

host:
> Sep 18 15:35:06 homeassistant systemd[1]: Mounting Supervisor cifs mount: SSD...
> Sep 18 15:35:06 homeassistant mount[3454102]: error 2 (No such file or directory) opening credential file /mnt/data/supervisor/.mounts_credentials/SSD
> Sep 18 15:35:06 homeassistant systemd[1]: mnt-data-supervisor-mounts-SSD.mount: Mount process exited, code=exited, status=2/INVALIDARGUMENT
> Sep 18 15:35:06 homeassistant systemd[1]: mnt-data-supervisor-mounts-SSD.mount: Failed with result 'exit-code'.
> Sep 18 15:35:06 homeassistant systemd[1]: Failed to mount Supervisor cifs mount: SSD.

The only way i found to fix this error was by creating that credential file manually. Previously when i had problems with the drive i deleted the mount settings from the Home Assistant network storage page and restartet SambaNAS and then it recreated the entry, but now there is no mount listed.

### Reproduction steps

```bash
1. stop SambaNAS when disk crashes
2. remove external disk and repair on windows machine
3. reconnect disk to HA (Raspberry PI 4) 
4. start SambaNAS
```

### Addon Logs

```bash
Wait Samba Server to going up..(max 30s)
[13:57:29] ERROR: Got unexpected response from the API: Reloading SSD did not succeed. Check host logs for errors from mount or systemd unit mnt-data-supervisor-mounts-SSD.mount for details.
s6-rc: warning: unable to start service cifs-supervisor-mount: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container.
s6-rc: info: service init-smartd: stopping
s6-rc: info: service init-mqtt: stopping
s6-rc: info: service wsdd: stopping
s6-rc: info: service smbd: stopping
s6-rc: info: service nmbd: stopping
s6-rc: info: service hd-idle: stopping
s6-rc: info: service avahi: stopping
s6-rc: info: service init-smartd successfully stopped
s6-rc: info: service init-mqtt successfully stopped
Got SIGTERM: going down...
Got SIGTERM, quitting.
s6-rc: info: service wsdd successfully stopped
s6-rc: info: service avahi successfully stopped
s6-rc: info: service smbd successfully stopped
s6-rc: info: service nmbd successfully stopped
s6-rc: info: service init-samba: stopping
s6-rc: info: service init-samba successfully stopped
s6-rc: info: service hd-idle successfully stopped
s6-rc: info: service init-automount: stopping
s6-rc: info: service init-automount successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service base-addon-log-level: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service base-addon-log-level successfully stopped
s6-rc: info: service base-addon-banner: stopping
s6-rc: info: service base-addon-banner successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
```

### Addon Config

```bash
workgroup: WORKGROUP
username: homeassistant
allow_hosts:
  - 10.0.0.0/8
  - 172.16.0.0/12
  - 192.168.0.0/16
  - 169.254.0.0/16
  - fe80::/10
  - fc00::/7
automount: true
moredisks: []
available_disks_log: true
medialibrary:
  enable: true
veto_files:
  - ._*
  - .DS_Store
  - Thumbs.db
  - icon?
  - .Trashes
compatibility_mode: false
wsdd2: false
mqtt_use_legacy_entities: false
autodiscovery: {}
other_users: []
acl: []
interfaces: []
password: *****
log_level: info
```

### Architecture

armv7

### OS

HaOS

**Comment by dianlight** [comment]
I need the startup log of the addon to understand the problem. Can you integrate te issu?

**Comment by tr4nt0r** [comment]
> I need the startup log of the addon to understand the problem. Can you integrate te issu?

Sorry, i don't have the addon logs anymore. Tried to find them on the host but the logrotation already deleted them. I would have to wait till the NTFS partition crushes again but i really hope it doesn't happen.

I tried to understand what happens and found from where the error is thrown

https://github.com/home-assistant/supervisor/blob/042921925dfb8e0b794df4ebaf663116d7f60f73/supervisor/mounts/mount.py#L286

**Comment by tr4nt0r** [comment]
So it happened again. Edited the issue and added the Addon Logs as requested.

**Comment by tr4nt0r** [comment]
Here is some more related logs from the host system. This is the shutdown process. Only thing i did was restart the addon, before everything was working normal.


```
Oct 04 11:57:04 homeassistant addon_1a32f091_sambanas[606]: [13:57:04] INFO: MQTT disk cleanup Done.Oct 04 11:57:04 homeassistant addon_1a32f091_sambanas[606]: [13:57:04] INFO: mqtt-handler stoppedOct 04 11:57:04 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service mqtt-disk-handler successfully stoppedOct 04 11:57:04 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service init-mqtt: stoppingOct 04 11:57:04 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service cifs-supervisor-mount: stopping
Oct 04 11:57:04 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service init-mqtt successfully stoppedOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: [13:57:05] INFO: Umount Network Automount Shares...Oct 04 11:57:05 homeassistant hassio_supervisor[606]: 23-10-04 13:57:05 INFO (MainThread) [supervisor.api.middleware.security] /mounts/SSD access from 1a32f091_sambanasOct 04 11:57:05 homeassistant hassio_supervisor[606]: 23-10-04 13:57:05 INFO (MainThread) [supervisor.mounts.manager] Removing mount: SSD
Oct 04 11:57:05 homeassistant systemd[1]: Unmounting Supervisor bind mount: bind_SSD...Oct 04 11:57:05 homeassistant umount[3181251]: umount: /mnt/data/supervisor/media/SSD: target is busy.Oct 04 11:57:05 homeassistant systemd[1]: mnt-data-supervisor-media-SSD.mount: Mount process exited, code=exited, status=32/n/aOct 04 11:57:05 homeassistant systemd[1]: Failed unmounting Supervisor bind mount: bind_SSD.
Oct 04 11:57:05 homeassistant systemd[1]: Unmounting Supervisor bind mount: bind_SSD...Oct 04 11:57:05 homeassistant umount[3181252]: umount: /mnt/data/supervisor/media/SSD: target is busy.Oct 04 11:57:05 homeassistant systemd[1]: mnt-data-supervisor-media-SSD.mount: Mount process exited, code=exited, status=32/n/aOct 04 11:57:05 homeassistant systemd[1]: Failed unmounting Supervisor bind mount: bind_SSD.Oct 04 11:57:05 homeassistant systemd[1]: Unmounting Supervisor cifs mount: SSD...Oct 04 11:57:05 homeassistant systemd[1]: mnt-data-supervisor-mounts-SSD.mount: Deactivated successfully.Oct 04 11:57:05 homeassistant systemd[1]: Unmounted Supervisor cifs mount: SSD.
Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: [13:57:05] INFO: Return from Umount Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service cifs-supervisor-mount successfully stoppedOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service wsdd: stoppingOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service smbd: stoppingOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service nmbd: stoppingOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: 2023-10-04 13:57:05,819:wsdd INFO(pid 349): received termination/interrupt signal, tearing downOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: Server exit (termination signal)
Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: Got SIGTERM: going down...Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: send_local_master_announcement: type 0 for name HOMEASSISTANT on subnet 192.168.178.59 for workgroup WORKGROUPOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: send_host_announcement: type 0 for host HOMEASSISTANT on subnet 192.168.178.59 for workgroup WORKGROUPOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: send_local_master_announcement: type 0 for name HOMEASSISTANT on subnet 172.30.32.1 for workgroup WORKGROUP
Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: send_host_announcement: type 0 for host HOMEASSISTANT on subnet 172.30.32.1 for workgroup WORKGROUPOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: 2023-10-04 13:57:05,830:wsdd INFO(pid 349): shutting down gracefully...Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: Server exit (termination signal)Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: homeassistant (ipv4:192.168.178.59:54866) closed connection to service SSD
Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: 2023-10-04 13:57:05,848:wsdd INFO(pid 349): scheduling Bye message via eth0 to ff02::c%eth0
Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: homeassistant (ipv4:192.168.178.59:54866) closed connection to service IPC$Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: chdir_below_conn: Bad access attempt: . is a symlink outside the share pathOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: conn_rootdir =/tmpOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: resolved_name=/Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: Server exit (termination signal)
Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: [13:57:05] INFO: smbd stoppedOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service smbd successfully stoppedOct 04 11:57:06 homeassistant addon_1a32f091_sambanas[606]: [13:57:06] INFO: nmbd stoppedOct 04 11:57:06 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service nmbd successfully stoppedOct 04 11:57:06 homeassistant addon_1a32f091_sambanas[606]: 2023-10-04 13:57:06,909:wsdd INFO(pid 349): Done.
Oct 04 11:57:07 homeassistant addon_1a32f091_sambanas[606]: [13:57:07] INFO: wsdd stoppedOct 04 11:57:07 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service wsdd successfully stoppedOct 04 11:57:07 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service init-samba: stoppingOct 04 11:57:07 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service init-samba successfully stoppedOct 04 11:57:07 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service init-automount: stopping
Oct 04 11:57:07 homeassistant hassio_supervisor[606]: 23-10-04 13:57:07 INFO (MainThread) [supervisor.api.middleware.security] /network/interface/default/info access from 1a32f091_sambanasOct 04 11:57:08 homeassistant hassio_supervisor[606]: 23-10-04 13:57:08 INFO (MainThread) [supervisor.api.middleware.security] /network/interface/eth0/info access from 1a32f091_sambanasOct 04 11:57:08 homeassistant addon_1a32f091_sambanas[606]: [13:57:08] INFO: Unmount drivers:Oct 04 11:57:08 homeassistant addon_1a32f091_sambanas[606]: /mnt/SSDOct 04 11:57:08 homeassistant addon_1a32f091_sambanas[606]: [13:57:08] INFO: Bye.
Oct 04 11:57:08 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service init-automount successfully stopped
Oct 04 11:57:08 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service legacy-cont-init: stopping
Oct 04 11:57:08 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service legacy-cont-init successfully stopped
```

**Comment by dianlight** [comment]
Sorry but I need all log not only the after-error.  
Meantime try to go to Settings->System->Storage and check if under `Network Sorage` there is your share delete it ant try to restart the addon.

**Comment by dianlight** [comment]
> Here is some more related logs from the host system. This is the shutdown process. Only thing i did was restart the addon, before everything was working normal.
> 
> ```
> Oct 04 11:57:04 homeassistant addon_1a32f091_sambanas[606]: [13:57:04] INFO: MQTT disk cleanup Done.Oct 04 11:57:04 homeassistant addon_1a32f091_sambanas[606]: [13:57:04] INFO: mqtt-handler stoppedOct 04 11:57:04 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service mqtt-disk-handler successfully stoppedOct 04 11:57:04 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service init-mqtt: stoppingOct 04 11:57:04 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service cifs-supervisor-mount: stopping
> Oct 04 11:57:04 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service init-mqtt successfully stoppedOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: [13:57:05] INFO: Umount Network Automount Shares...Oct 04 11:57:05 homeassistant hassio_supervisor[606]: 23-10-04 13:57:05 INFO (MainThread) [supervisor.api.middleware.security] /mounts/SSD access from 1a32f091_sambanasOct 04 11:57:05 homeassistant hassio_supervisor[606]: 23-10-04 13:57:05 INFO (MainThread) [supervisor.mounts.manager] Removing mount: SSD
> Oct 04 11:57:05 homeassistant systemd[1]: Unmounting Supervisor bind mount: bind_SSD...Oct 04 11:57:05 homeassistant umount[3181251]: umount: /mnt/data/supervisor/media/SSD: target is busy.Oct 04 11:57:05 homeassistant systemd[1]: mnt-data-supervisor-media-SSD.mount: Mount process exited, code=exited, status=32/n/aOct 04 11:57:05 homeassistant systemd[1]: Failed unmounting Supervisor bind mount: bind_SSD.
> Oct 04 11:57:05 homeassistant systemd[1]: Unmounting Supervisor bind mount: bind_SSD...Oct 04 11:57:05 homeassistant umount[3181252]: umount: /mnt/data/supervisor/media/SSD: target is busy.Oct 04 11:57:05 homeassistant systemd[1]: mnt-data-supervisor-media-SSD.mount: Mount process exited, code=exited, status=32/n/aOct 04 11:57:05 homeassistant systemd[1]: Failed unmounting Supervisor bind mount: bind_SSD.Oct 04 11:57:05 homeassistant systemd[1]: Unmounting Supervisor cifs mount: SSD...Oct 04 11:57:05 homeassistant systemd[1]: mnt-data-supervisor-mounts-SSD.mount: Deactivated successfully.Oct 04 11:57:05 homeassistant systemd[1]: Unmounted Supervisor cifs mount: SSD.
> Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: [13:57:05] INFO: Return from Umount Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service cifs-supervisor-mount successfully stoppedOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service wsdd: stoppingOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service smbd: stoppingOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service nmbd: stoppingOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: 2023-10-04 13:57:05,819:wsdd INFO(pid 349): received termination/interrupt signal, tearing downOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: Server exit (termination signal)
> Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: Got SIGTERM: going down...Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: send_local_master_announcement: type 0 for name HOMEASSISTANT on subnet 192.168.178.59 for workgroup WORKGROUPOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: send_host_announcement: type 0 for host HOMEASSISTANT on subnet 192.168.178.59 for workgroup WORKGROUPOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: send_local_master_announcement: type 0 for name HOMEASSISTANT on subnet 172.30.32.1 for workgroup WORKGROUP
> Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: send_host_announcement: type 0 for host HOMEASSISTANT on subnet 172.30.32.1 for workgroup WORKGROUPOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: 2023-10-04 13:57:05,830:wsdd INFO(pid 349): shutting down gracefully...Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: Server exit (termination signal)Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: homeassistant (ipv4:192.168.178.59:54866) closed connection to service SSD
> Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: 2023-10-04 13:57:05,848:wsdd INFO(pid 349): scheduling Bye message via eth0 to ff02::c%eth0
> Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: homeassistant (ipv4:192.168.178.59:54866) closed connection to service IPC$Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: chdir_below_conn: Bad access attempt: . is a symlink outside the share pathOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: conn_rootdir =/tmpOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: resolved_name=/Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: Server exit (termination signal)
> Oct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: [13:57:05] INFO: smbd stoppedOct 04 11:57:05 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service smbd successfully stoppedOct 04 11:57:06 homeassistant addon_1a32f091_sambanas[606]: [13:57:06] INFO: nmbd stoppedOct 04 11:57:06 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service nmbd successfully stoppedOct 04 11:57:06 homeassistant addon_1a32f091_sambanas[606]: 2023-10-04 13:57:06,909:wsdd INFO(pid 349): Done.
> Oct 04 11:57:07 homeassistant addon_1a32f091_sambanas[606]: [13:57:07] INFO: wsdd stoppedOct 04 11:57:07 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service wsdd successfully stoppedOct 04 11:57:07 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service init-samba: stoppingOct 04 11:57:07 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service init-samba successfully stoppedOct 04 11:57:07 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service init-automount: stopping
> Oct 04 11:57:07 homeassistant hassio_supervisor[606]: 23-10-04 13:57:07 INFO (MainThread) [supervisor.api.middleware.security] /network/interface/default/info access from 1a32f091_sambanasOct 04 11:57:08 homeassistant hassio_supervisor[606]: 23-10-04 13:57:08 INFO (MainThread) [supervisor.api.middleware.security] /network/interface/eth0/info access from 1a32f091_sambanasOct 04 11:57:08 homeassistant addon_1a32f091_sambanas[606]: [13:57:08] INFO: Unmount drivers:Oct 04 11:57:08 homeassistant addon_1a32f091_sambanas[606]: /mnt/SSDOct 04 11:57:08 homeassistant addon_1a32f091_sambanas[606]: [13:57:08] INFO: Bye.
> Oct 04 11:57:08 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service init-automount successfully stopped
> Oct 04 11:57:08 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service legacy-cont-init: stopping
> Oct 04 11:57:08 homeassistant addon_1a32f091_sambanas[606]: s6-rc: info: service legacy-cont-init successfully stopped
> ```

There are some process that are blocking your disk. Do you have other addons that read and write on he disk? Try to restart also this.

**Comment by tr4nt0r** [comment]
> Sorry but I need all log not only the after-error. Meantime try to go to Settings->System->Storage and check if under `Network Sorage` there is your share delete it ant try to restart the addon.

There is no share in the Network Storage section, i think it has been deleted when the samba nas addon shut down. 
![image](https://github.com/dianlight/hassio-addons/assets/4445816/d6178ef7-1558-4eb4-a472-2b22b0374314)

Thats maybe the reason why the credentials file is missing. But also i noticed the message reads "Reloading did not succeed" instead of "Mounting did not succeed". These are two different methods. 

https://github.com/home-assistant/supervisor/blob/042921925dfb8e0b794df4ebaf663116d7f60f73/supervisor/mounts/mount.py#L216

The mount method does recreate the credentials file, the reload method does not (as far as i could understand the code)

**Comment by dianlight** [comment]
> There is no share in the Network Storage section, i think it has been deleted when the samba nas addon shut down.

Correct. 

> Thats maybe the reason why the credentials file is missing.

Correct, the credential file is only internal to supervisor so delete the share delete also the related credential file.

>  But also i noticed the message reads "Reloading did not succeed" instead of "Mounting did not succeed". These are two different methods.

Yes but there are only one endpoint. I try to explain, form the addon I do only call to API to mount not to remount [Code](sambanas/rootfs/etc/s6-overlay/s6-rc.d/cifs-supervisor-mount/run ) the remount is triggered by an internal process of supervisor when you try to mount an already mounted device. The supervisor do remount  because the share is not fully unmounted because a process block it.

>  ``Oct 04 11:57:05 homeassistant systemd[1]: Unmounting Supervisor bind mount: bind_SSD...Oct 04 11:57:05 homeassistant umount[3181251]: umount: /mnt/data/supervisor/media/SSD: target is busy.Oct 04 11:57:05 homeassistant systemd[1]: mnt-data-supervisor-media-SSD.mount: Mount process exited, code=exited, status=32/n/aOct 04 11:57:05 homeassistant systemd[1]: Failed unmounting Supervisor bind mount: bind_SSD.
``

This is your problem: there is a process that stuck the mount and create the problem. ( for the supervisor is umounted but for the so level is mounted ).
I have successfull replicate the problem using a Plesk addon that index the share, if I try to umount ( stop the sambanas addon ) the drive is busy for the cache write and fail to umount. 
The shutdown process of an addon has a timeout so I can't block the request to stop to wait the end of write, I'm still looking for a solution or a patch to propose to the supervisor.

An host reboot resolve the problem?
What are the other addon or external client for the share? 


L.

**Comment by tr4nt0r** [comment]
Maybe it was Jellyfin which was still accessing the device. Last time i tried rebooting the host it did not resolve the problem, just created more problems (ntfs dirty flag). Only when i created the credentials file manually it reverted to a normal state.

**Comment by dianlight** [comment]
should be fixed  in the release 12.1.0-nas
